api/v1/model_handlers.go

```diff
@@ -336,84 +344,106 @@ func (a *Api) containerPaths(request *restful.Request, response *restful.Respons
 	response.WriteEntity(entities)
 }
 
-// allNodes returns a list of all the available node names in the cluster.
+// makeExternalEntityList converts the result of model's getX methods to the external type.
+func makeExternalEntityList(list []model_api.EntityListEntry) []types.ExternalEntityListEntry {
```

Comment:
it's actually an external (entity list), rather than an (external entity) list

This function converts a slice of internal entityListEntry to a slice of ExternalEntityListEntry.
The internal and external structs are identical, as of now, but they are distinct to avoid outputting internal types through the API